lib/cfn-model/parser/transform_registry.rb

```diff
@@ -0,0 +1,33 @@
+Dir["#{__dir__}/../transforms/*.rb"].each do |transform|
+  require "cfn-model/transforms/#{File.basename(transform, '.rb')}"
+end
+
+class CfnModel
+  # TransformRegistry provides a registry of CloudFormation transforms
+  # available for templates
+  class TransformRegistry
+    attr_reader :registry
+
+    def initialize
+      @registry = {
+        'AWS::Serverless-2016-10-31' => CfnModel::Transforms::Serverless
```

Comment:
remind me to talk about this one offline

Reply:
(done)